lib/solidus_support.rb

```diff
@@ -24,6 +24,18 @@ def payment_source_parent_class
       end
     end
 
+    def payment_method_parent_class(credit_card: false)
```

Comment:
Since this is only useful when `credit_card` is `true` (otherwise it is `Spree::PaymentMethod` in all versions).

Maybe we should call this `credit_card_payment_method_parent_class` instead, and only deal with that case?

Either way :+1: